
project onos-apps/apps/vtn/
diff --git a/xos/synchronizer/vtn_synchronizer_config b/xos/synchronizer/vtn_synchronizer_config
index dad5cb6..f317dee 100644
--- a/xos/synchronizer/vtn_synchronizer_config
+++ b/xos/synchronizer/vtn_synchronizer_config
@@ -33,6 +33,7 @@ driver=openstack
 pretend=False
 backoff_disabled=True
 enable_watchers=True
+logstash_hostport=10.100.198.222:5617
 
 [nova]
 ca_ssl_cert=/etc/ssl/certs/ca-certificates.crt

project orchestration/service-profile/
diff --git a/cord-pod-ansible/files/monitoring_synchronizer_config b/cord-pod-ansible/files/monitoring_synchronizer_config
index 1ee96f0..932a32b 100644
--- a/cord-pod-ansible/files/monitoring_synchronizer_config
+++ b/cord-pod-ansible/files/monitoring_synchronizer_config
@@ -37,6 +37,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'
diff --git a/cord-pod-ansible/files/vcpe_synchronizer_config b/cord-pod-ansible/files/vcpe_synchronizer_config
index 9da6ede..9ef259b 100644
--- a/cord-pod-ansible/files/vcpe_synchronizer_config
+++ b/cord-pod-ansible/files/vcpe_synchronizer_config
@@ -38,6 +38,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True
diff --git a/cord-pod-ansible/files/vtr_synchronizer_config b/cord-pod-ansible/files/vtr_synchronizer_config
index 223ab00..4e3008c 100644
--- a/cord-pod-ansible/files/vtr_synchronizer_config
+++ b/cord-pod-ansible/files/vtr_synchronizer_config
@@ -38,6 +38,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True
diff --git a/cord-pod/files/monitoring_synchronizer_config b/cord-pod/files/monitoring_synchronizer_config
index cfd73a0..485f993 100644
--- a/cord-pod/files/monitoring_synchronizer_config
+++ b/cord-pod/files/monitoring_synchronizer_config
@@ -37,6 +37,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'
diff --git a/cord-pod/files/openstack_synchronizer_config b/cord-pod/files/openstack_synchronizer_config
index 3b7f8e9..29077e2 100644
--- a/cord-pod/files/openstack_synchronizer_config
+++ b/cord-pod/files/openstack_synchronizer_config
@@ -22,6 +22,7 @@ save_ansible_output=True
 backoff_disabled=True
 pretend=False
 images_directory=/opt/xos/images
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'
diff --git a/cord-pod/files/vcpe_synchronizer_config b/cord-pod/files/vcpe_synchronizer_config
index babe94f..34e0f2b 100644
--- a/cord-pod/files/vcpe_synchronizer_config
+++ b/cord-pod/files/vcpe_synchronizer_config
@@ -39,6 +39,7 @@ proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
 enable_watchers=True
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True
diff --git a/cord-pod/files/vtr_synchronizer_config b/cord-pod/files/vtr_synchronizer_config
index 3d3209c..56af7fc 100644
--- a/cord-pod/files/vtr_synchronizer_config
+++ b/cord-pod/files/vtr_synchronizer_config
@@ -38,6 +38,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True
diff --git a/mcord/files/monitoring_synchronizer_config b/mcord/files/monitoring_synchronizer_config
index cfd73a0..485f993 100644
--- a/mcord/files/monitoring_synchronizer_config
+++ b/mcord/files/monitoring_synchronizer_config
@@ -37,6 +37,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/root/setup/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'
diff --git a/mcord/files/openstack_synchronizer_config b/mcord/files/openstack_synchronizer_config
index 82c2f79..58dece3 100644
--- a/mcord/files/openstack_synchronizer_config
+++ b/mcord/files/openstack_synchronizer_config
@@ -22,6 +22,7 @@ save_ansible_output=True
 backoff_disabled=False
 pretend=False
 images_directory=/opt/xos/images
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'
diff --git a/opencloud/files/openstack_synchronizer_config b/opencloud/files/openstack_synchronizer_config
index 74d68e3..8b0f8f9 100644
--- a/opencloud/files/openstack_synchronizer_config
+++ b/opencloud/files/openstack_synchronizer_config
@@ -22,6 +22,7 @@ save_ansible_output=True
 backoff_disabled=True
 pretend=False
 images_directory=/opt/xos/images
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos/
diff --git a/xos/synchronizers/onboarding/onboarding_synchronizer_config b/xos/synchronizers/onboarding/onboarding_synchronizer_config
index fb33044..6877378 100644
--- a/xos/synchronizers/onboarding/onboarding_synchronizer_config
+++ b/xos/synchronizers/onboarding/onboarding_synchronizer_config
@@ -32,4 +32,4 @@ backoff_disabled=True
 pretend=False
 save_ansible_output=True
 node_key=/opt/cord_profile/node_key
-
+logstash_hostport=10.100.198.222:5617
diff --git a/xos/synchronizers/syndicate/syndicate_synchronizer_config b/xos/synchronizers/syndicate/syndicate_synchronizer_config
index 7c9d2d2..c9f3cf5 100644
--- a/xos/synchronizers/syndicate/syndicate_synchronizer_config
+++ b/xos/synchronizers/syndicate/syndicate_synchronizer_config
@@ -29,6 +29,7 @@ steps_dir=/opt/xos/synchronizers/syndicate/steps
 deleters_dir=/opt/xos/synchronizers/syndicate/deleters
 log_file=console
 driver=None
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/fabric/
diff --git a/xos/synchronizer/fabric_synchronizer_config b/xos/synchronizer/fabric_synchronizer_config
index e98abe5..b15fcc3 100644
--- a/xos/synchronizer/fabric_synchronizer_config
+++ b/xos/synchronizer/fabric_synchronizer_config
@@ -21,3 +21,4 @@ pretend=False
 backoff_disabled=True
 save_ansible_output=True
 proxy_ssh=False
+logstash_hostport=10.100.198.222:5617

project orchestration/xos_services/globalxos/
diff --git a/xos/synchronizer/globalxos_synchronizer_config b/xos/synchronizer/globalxos_synchronizer_config
index f06031e..86ae9cc 100644
--- a/xos/synchronizer/globalxos_synchronizer_config
+++ b/xos/synchronizer/globalxos_synchronizer_config
@@ -22,7 +22,7 @@ save_ansible_output=True
 backoff_disabled=True
 pretend=False
 images_directory=/opt/xos/images
-
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/hypercache/
diff --git a/xos/synchronizer/hpc_synchronizer_config b/xos/synchronizer/hpc_synchronizer_config
index 9d4e70a..015cd77 100644
--- a/xos/synchronizer/hpc_synchronizer_config
+++ b/xos/synchronizer/hpc_synchronizer_config
@@ -30,6 +30,7 @@ log_file=console
 #/var/log/hpc.log
 driver=None
 #cmi_hostname=openclouddev0.internet2.edu
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/metro-net/
diff --git a/xos/synchronizer/metronetwork_synchronizer_config b/xos/synchronizer/metronetwork_synchronizer_config
index 86847b9..99de62a 100644
--- a/xos/synchronizer/metronetwork_synchronizer_config
+++ b/xos/synchronizer/metronetwork_synchronizer_config
@@ -32,6 +32,7 @@ driver=None
 pretend=False
 backoff_disabled=True
 fofum_disabled=True
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/monitoring/
diff --git a/xos/synchronizer/monitoring_synchronizer_config b/xos/synchronizer/monitoring_synchronizer_config
index 5cfe5e3..c30361e 100644
--- a/xos/synchronizer/monitoring_synchronizer_config
+++ b/xos/synchronizer/monitoring_synchronizer_config
@@ -35,6 +35,7 @@ save_ansible_output=True
 # set proxy_ssh to false on cloudlab
 proxy_ssh=False
 full_setup=True
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/onos-service/
diff --git a/xos/synchronizer/onos_synchronizer_config b/xos/synchronizer/onos_synchronizer_config
index c7a2f2c..a7f48ba 100644
--- a/xos/synchronizer/onos_synchronizer_config
+++ b/xos/synchronizer/onos_synchronizer_config
@@ -35,6 +35,7 @@ save_ansible_output=True
 # set proxy_ssh to false on cloudlab
 proxy_ssh=False
 full_setup=True
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/openstack/
diff --git a/xos/synchronizer/openstack_synchronizer_config b/xos/synchronizer/openstack_synchronizer_config
index 3b7f8e9..29077e2 100644
--- a/xos/synchronizer/openstack_synchronizer_config
+++ b/xos/synchronizer/openstack_synchronizer_config
@@ -22,6 +22,7 @@ save_ansible_output=True
 backoff_disabled=True
 pretend=False
 images_directory=/opt/xos/images
+logstash_hostport=10.100.198.222:5617
 
 [feefie]
 client_id='vicci_dev_central'

project orchestration/xos_services/vsg/
diff --git a/xos/synchronizer/vsg_synchronizer_config b/xos/synchronizer/vsg_synchronizer_config
index fd254a2..2a12003 100644
--- a/xos/synchronizer/vsg_synchronizer_config
+++ b/xos/synchronizer/vsg_synchronizer_config
@@ -39,6 +39,7 @@ proxy_ssh=True
 proxy_ssh_key=/opt/cord_profile/node_key
 proxy_ssh_user=root
 enable_watchers=True
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True

project orchestration/xos_services/vtr/
diff --git a/xos/synchronizer/vtn_vtr_synchronizer_config b/xos/synchronizer/vtn_vtr_synchronizer_config
index 2c9140a..a10e28b 100644
--- a/xos/synchronizer/vtn_vtr_synchronizer_config
+++ b/xos/synchronizer/vtn_vtr_synchronizer_config
@@ -38,6 +38,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/opt/xos/synchronizers/vtr/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True
diff --git a/xos/synchronizer/vtr_synchronizer_config b/xos/synchronizer/vtr_synchronizer_config
index a8db7ed..4743d65 100644
--- a/xos/synchronizer/vtr_synchronizer_config
+++ b/xos/synchronizer/vtr_synchronizer_config
@@ -38,6 +38,7 @@ full_setup=True
 proxy_ssh=True
 proxy_ssh_key=/opt/cord_profile/node_key
 proxy_ssh_user=root
+logstash_hostport=10.100.198.222:5617
 
 [networking]
 use_vtn=True
